package.json

```diff
@@ -23,37 +23,37 @@
     "node": ">=0.10.0"
   },
   "dependencies": {
-    "URIjs": "1.14.1",
-    "bluebird": "2.3.10",
+    "URIjs": "^1.14.2",
```

Comment:
No, I'm the stupid one. You're thinking of @jrgm (the smart one).
I thought we did `npm shrinkwrap --dev` in all the repos. We should probably standardize on one method and use it consistently across projects.